### ISSUE TITLE
[Features] Integration with other 3rd party sites

### ISSUE THREAD

**Issue by JetScalawag** [master-technology/AssetManagerStudio#64]
Please include the ability to be alerted for discounted Unreal assets appearing in Humble Bundles, Fanatical, and the like

**Comment by JetScalawag** [comment]
Ach what am I thinking? I'm mixing up this Asset Manager with Fab itself! Well, that's because its a lot easier to use than Fab.
Perhaps this app can be a full-on replacement browser for Unreal assets in Fab? Minus the purchasing functionality, of course.
If this happens, please consider the above suggestion!

**Comment by NathanaelA** [comment]
I actually liked this idea, however it is for the server side stuff I plan on doing in the future -- at that point I'll have the stuff to pull data from other sites to track 3rd party sales events.

**Comment by JetScalawag** [comment]
I'm excited that you're actually considering making the app capable of browsing new assets from Fab, with awareness of Humble Bundle, Fanatical, etc. offers. 
To be honest, I thought I was going too far with this request. But hey, please don't let me stop you!

**Comment by NathanaelA** [comment]
No, I have (what feels like) a bazillion assets from other sites, I really want to integrate the ability to have them easily accessible in AMS.    Adding the ability to see sales events is just a small step farther down the road of where I wanted to go already.  :grinning:

**Comment by JetScalawag** [comment]
Love it! Hey Epic Megagrants do you accept nominations?

**Comment by NathanaelA** [comment]
I tried last year, no go.   But wouldn't hurt if others suggested it.  :grinning:
